http-clients/src/main/java/org/triplea/http/client/HttpConstants.java

```diff
@@ -0,0 +1,15 @@
+package org.triplea.http.client;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+/**
+ * Utility class with constants for HTTP clients.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class HttpConstants {
+  public static final String ACCEPT_JSON = "Accept: application/json";
+  public static final String ACCEPT_TEXT = "Accept: text/plain";
+  public static final String CONTENT_TYPE_JSON = "Content-Type: application/json";
+  public static final String CONTENT_TYPE_TEXT = "Content-Type: text/plain";
+}
```

Comment:
I'm surprised that there isn't a built-in way to do this.
The apache HTTPComponents library has constants for the most headers and common values